assignments/mini-libc/libc/include/stdlib.h

```diff
@@ -0,0 +1,27 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
+#ifndef __STDLIB_H__
+#define __STDLIB_H__	1
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <internal/types.h>
+#include <internal/essentials.h>
+
+#define EXIT_SUCCESS	0
+#define EXIT_FAILURE	1
+
+void *malloc(size_t size);
+void free(void *ptr);
+void *calloc(size_t nmemb, size_t size);
+void *realloc(void *ptr, size_t size);
+void *reallocarray(void *ptr, size_t nmemb, size_t size);
+long exit(long exit_code);
```

Comment:
Any reasons to not do these lines as TODOs?

Reply:
Why would you want them to be TODOs? They just specify the API. Without them, the checker would spit out warnings about calling unknown functions.

Reply:
What @teodutu said.

Reply:
Ok, but for `puts` it's a TODO.